include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -1579,8 +1579,14 @@ namespace VectorTools
                 const Quadrature<dim-1>        &q_boundary,
                 const bool                      project_to_boundary_first)
   {
+#ifdef _MSC_VER
+    Assert(false, 
+           ExcMessage("Please specify the mapping explicitly "
+                      "when building with MSVC!"));    
```

Comment:
Oh, you can't find a way at all to call the other function? Not even if you locally declare a static variable for the mapping?

Reply:
I didn't have the chance to do more than compile the library yet. All I can say is that using this particular line ends in an ICE.

Reply:
Shame. But I'd say let's go with it. If at one point we have things working on a regular basis and with a regular tester, we can think of a better way.

Unless, of course, you want to try the obvious alternative
```
#ifdef _MSC_VER
 static MappingQ1<dim,spacedim> mapping;
 project(mapping, dof, constraints, quadrature, function, vec,
             enforce_zero_boundary, q_boundary, project_to_boundary_first);
#else
```
and see whether that works.

Reply:
That I already tried without success. :confused:

Reply:
Ah, too bad. It sucks to have compilers so broken that even the obvious replacements don't work :-(

Never mind then, let's just go with this.